### PR TITLE
feat: add `nuxt-module-hotjar`

### LINKED ISSUE #861: [Module Listing Request]: `nuxt-module-hotjar`
### Description

This module integrates Hotjar script implementation with Nuxt 3. This module allows easy integration of Hotjar into a Nuxt project.

### Repository

https://github.com/damevin/nuxt-module-hotjar

### npm

https://www.npmjs.com/package/nuxt-module-hotjar

### Nuxt Compatibility

Nuxt 3

### PR DESCRIPTION
**Pull request by damevin**
Resolves #861

### CODE REVIEW

File: modules/nuxt-module-hotjar.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: nuxt-hotjar
+description: Add HotJar to your Nuxt.js application
+repo: damevin/nuxt-module-hotjar
+npm: nuxt-nodule-hotjar
```
Comment (danielroe) [inline modules/nuxt-module-hotjar.yml:4]:
```suggestion
npm: nuxt-module-hotjar
```

**Review by danielroe — COMMENTED** [review]
Would you delete one of these?

**Comment by damevin** [comment]
@danielroe no it's ready !

**Comment by damevin** [comment]
@danielroe fixed ! sorry 😅

**Pull request merged**